packages/server/src/sdk/app/rows/tests/internal.spec.ts

```diff
@@ -199,7 +199,7 @@ describe("sdk >> rows >> internal", () => {
           await internalSdk.save(table._id!, row, config.getUser()._id)
         }
         await Promise.all(
-          makeRows(10).map(row =>
+          makeRows(20).map(row =>
```

Comment:
Decided to up the number of rows to be a bit more sure we're avoiding conflicts.